code/graphics/2d.cpp

```diff
@@ -1731,6 +1731,18 @@ void gr_get_color( int *r, int *g, int *b )
 	if (b) *b = gr_screen.current_color.blue;
 }
 
+// clamps a color value to 0-255
+void gr_clamp_color_value(int &val)
```

Comment:
We should have a macro `#define` to do clamping, but I haven't seen how often this function is called to say if it warrents yet another clamping function.

Reply:
Coincidentally I ran into that macro just this morning. Didn't realize it existed when I whipped this up. I have changed the code to use the macro instead.